`cfg_match` macro name conflict on the latest nightly

Trying to compile this with the latest nightly, I got these errors:
```rs
error[E0658]: use of unstable library feature 'cfg_match'
  --> /home/imbris/.cargo/registry/src/index.crates.io-6f17d22bba15001f/maybe-dangling-0.1.0/src/maybe_dangling.rs:65:1
   |
65 | crate::cfg_match! {
   | ^^^^^^^^^^^^^^^^
   |
   = note: see issue #115585 <https://github.com/rust-lang/rust/issues/115585> for more information
   = help: add `#![feature(cfg_match)]` to the crate attributes to enable

error: no rules expected the token `feature`
  --> /home/imbris/.cargo/registry/src/index.crates.io-6f17d22bba15001f/maybe-dangling-0.1.0/src/maybe_dangling.rs:66:5
   |
66 |     feature = "nightly-dropck_eyepatch" => {
   |     ^^^^^^^ no rules expected this token in macro call
   |
note: while trying to match `cfg`
  --> /rustc/6cf088810f66fff15d05bf7135c5f5888b7c93b4/library/core/src/macros/mod.rs:204:11

error[E0659]: `cfg_match` is ambiguous
  --> /home/imbris/.cargo/registry/src/index.crates.io-6f17d22bba15001f/maybe-dangling-0.1.0/src/lib.rs:42:5
   |
42 | use cfg_match;
   |     ^^^^^^^^^ ambiguous name
   |
   = note: ambiguous because of a conflict between a macro-expanded name and a less macro-expanded name from outer scope during import or macro resolution
   = note: `cfg_match` could refer to a macro from prelude
note: `cfg_match` could also refer to the macro defined here
  --> /home/imbris/.cargo/registry/src/index.crates.io-6f17d22bba15001f/maybe-dangling-0.1.0/src/lib.rs:17:1
   |
17 | / macro_rules! cfg_match {
18 | |     (
19 | |         _ => { $($expansion:tt)* } $(,)?
20 | |     ) => (
...  |
40 | |     });
41 | | }
   | |_^

error[E0658]: use of unstable library feature 'cfg_match'
  --> /home/imbris/.cargo/registry/src/index.crates.io-6f17d22bba15001f/maybe-dangling-0.1.0/src/lib.rs:42:5
   |
42 | use cfg_match;
   |     ^^^^^^^^^
   |
   = note: see issue #115585 <https://github.com/rust-lang/rust/issues/115585> for more information
   = help: add `#![feature(cfg_match)]` to the crate attributes to enable
```

The fix made here looks promising: https://github.com/getditto/safer_ffi/pull/191

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -11,7 +11,7 @@ mod manually_drop;
 
 #[rustfmt::skip]
 /// I really don't get the complexity of `cfg_if!`…
-macro_rules! cfg_match {
+macro_rules! match_cfg {
     (
         _ => { $($expansion:tt)* } $(,)?
     ) => (
@@ -23,17 +23,17 @@ macro_rules! cfg_match {
         $($($rest:tt)+)? )?
     ) => (
         #[cfg($cfg)]
-        crate::cfg_match! { _ => $expansion } $($(
+        crate::match_cfg! { _ => $expansion } $($(
 
         #[cfg(not($cfg))]
-        crate::cfg_match! { $($rest)+ } )?)?
+        crate::match_cfg! { $($rest)+ } )?)?
     );
 
-    // Bonus: expression-friendly syntax: `cfg_match!({ … })`
+    // Bonus: expression-friendly syntax: `match_cfg!({ … })`
     ({
         $($input:tt)*
     }) => ({
-        crate::cfg_match! { $($input)* }
+        crate::match_cfg! { $($input)* }
     });
 }
-use cfg_match;
+use match_cfg;

--- a/src/maybe_dangling.rs
+++ b/src/maybe_dangling.rs
@@ -292,7 +292,7 @@ impl<T> MaybeDangling<T> {
 }
 
 // The main difference with `ManuallyDrop`: automatic drop glue!
-crate::cfg_match! {
+crate::match_cfg! {
     feature = "nightly-dropck_eyepatch" => {
         #[allow(unsafe_code)]
         unsafe impl<#[may_dangle] T> Drop for MaybeDangling<T> {

